.github/workflows/ci.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
     - uses: reviewdog/action-misspell@v1
       with:
         exclude: ./.github/workflows/*
-        ignore: creater
+        ignore: creater,importas
```

Comment:
We can also just replace `misspell`: https://golangci-lint.run/usage/linters/#misspell -- does golangci-lint try to lint its own config? 🤔

Reply:
I had that thought ...